<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(41, 41, 41);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame { 
	background-color: rgb(85, 85, 127);

border-color:green;

border-radius:15px;


}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_90derece">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{

   color: white;
	background-color: blue;
border-radius: 9px;
padding: 5px;
    box-shadow: 5px 5px 5px 5px rgba(0, 0, 0, 50);



}
QPushButton:pressed {

padding-left: 10px;
padding-top: 10px;
}
</string>
     </property>
     <property name="text">
      <string>90°</string>
     </property>
     <property name="icon">
      <iconset resource="Iconlar.qrc">
       <normaloff>:/Iconlar/Iconlar/play-button-arrowhead.png</normaloff>:/Iconlar/Iconlar/play-button-arrowhead.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_45derece">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{

   color: white;
	background-color: blue;
border-radius: 9px;
padding: 5px;
    box-shadow: 5px 5px 5px 5px rgba(0, 0, 0, 50);



}
QPushButton:pressed {

padding-left: 10px;
padding-top: 10px;
}
</string>
     </property>
     <property name="text">
      <string> 45°</string>
     </property>
     <property name="icon">
      <iconset resource="Iconlar.qrc">
       <normaloff>:/Iconlar/Iconlar/play-button-arrowhead.png</normaloff>:/Iconlar/Iconlar/play-button-arrowhead.png</iconset>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ok">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{

   color: white;
	background-color: blue;
border-radius: 9px;
padding: 5px;
    box-shadow: 5px 5px 5px 5px rgba(0, 0, 0, 50);



}
QPushButton:pressed {

padding-left: 10px;
padding-top: 10px;
}
</string>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
     <property name="icon">
      <iconset resource="Iconlar.qrc">
       <normaloff>:/Iconlar/Iconlar/play-button-arrowhead.png</normaloff>:/Iconlar/Iconlar/play-button-arrowhead.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;
color:dark-black;
border:3px-black;
font:20pt;
font-weight:1200;
border:3px-black;
border-radius:0px;
</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{

   color: white;
	background-color: blue;
border-radius: 9px;
padding: 5px;
    box-shadow: 5px 5px 5px 5px rgba(0, 0, 0, 50);



}
QPushButton:pressed {

padding-left: 10px;
padding-top: 10px;
}
</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_der_gosterge">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;
color:dark-black;
border:3px-black;
font:20pt;
font-weight:1200;
border:3px-black;
border-radius:0px;
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_0derece">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{

   color: white;
	background-color: blue;
border-radius: 9px;
padding: 5px;
    box-shadow: 5px 5px 5px 5px rgba(0, 0, 0, 50);



}
QPushButton:pressed {

padding-left: 10px;
padding-top: 10px;
}
</string>
     </property>
     <property name="text">
      <string>0°</string>
     </property>
     <property name="icon">
      <iconset resource="Iconlar.qrc">
       <normaloff>:/Iconlar/Iconlar/play-button-arrowhead.png</normaloff>:/Iconlar/Iconlar/play-button-arrowhead.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:center;
font: 900 18pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>SERVO MOTOR</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>0</y>
      <width>781</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color: rgb(85, 85, 127);
font-weight: bold;
text-font:25pt;
text-align:center;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>681</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:center;
font: 900 italic 36pt &quot;Segoe UI&quot;;
color:dark-black;
font-weight: bold;
font: 20pt &quot;Swis721 BlkEx BT&quot;;</string>
     </property>
     <property name="text">
      <string>IZLEME VE KONTROL PANELI</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>541</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color: rgb(85, 85, 127);
border-radius:8px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>521</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius:5px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_terminal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-color:dark-black;
text-align:center;
font:14pt;
color:#030303;
font-weight: bold;
</string>
     </property>
     <property name="text">
      <string>BILGILENDIRME TERMINALI</string>
     </property>
    </widget>
    <widget class="QLabel" name="saat_gostergesi">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 800 9pt &quot;Rockwell Extra Bold&quot;;
background-color:black;
color:white;</string>
     </property>
     <property name="text">
      <string>Clock</string>
     </property>
    </widget>
    <widget class="QLabel" name="zaman_gostergesi">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>38</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 800 9pt &quot;Rockwell Extra Bold&quot;;
background-color:black;
color:white;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color: rgb(85, 85, 127);
border-radius:8px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:center;
font: 900 18pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string>HAZNE 1 </string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:10px;
background-color:white;
text-align:center;
font: 700 16pt &quot;Segoe UI&quot;;
border-radius:0px;
color:blue;
</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color: rgb(85, 85, 127);
border-radius:8px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: blue;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:center;
font: 900 14pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="text">
      <string>SICAKLIK  °C</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:center;
font: 900 18pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="text">
      <string>NEM  %</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_nem">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: blue;
color:white;</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(85, 85, 127);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="startBlinking">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(85, 85, 127);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="durdurma">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:red;
border-radius:10px;
font: 12pt &quot;Segoe MDL2 Assets&quot;;
color:dark-black;
text-align:center;
font-weight: bold;
</string>
     </property>
     <property name="text">
      <string>GAZ ALGILANDI</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>71</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:dark-black;
color:white;
font-weight: bold;
font-size:10pt;
text-align: top;
</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:red;
border-radius:10px;
font: 13pt &quot;Segoe MDL2 Assets&quot;;
text-align:center;
color:dark-black;
font-weight: bold;








</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>     ! YANGIN !</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startBlinking2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="durdurma2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_7">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color: rgb(85, 85, 127);
border-radius:8px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 13pt &quot;Arial Black&quot;;</string>
      </property>
      <property name="text">
       <string>Voltaj (V)</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_volt">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>64</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: blue;</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLCDNumber" name="lcdNumber_Akim">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>64</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: blue;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>16</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 13pt &quot;Arial Black&quot;;</string>
      </property>
      <property name="text">
       <string>Akım (A)</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLCDNumber" name="lcdNumber_Guc">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>64</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: blue;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>9</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 13pt &quot;Arial Black&quot;;</string>
      </property>
      <property name="text">
       <string>Güç  (W)</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Iconlar.qrc"/>
 </resources>
 <connections/>
</ui>
